python/dgl/graph_serialize.py

```diff
@@ -0,0 +1,63 @@
+"""For Graph Serialization"""
+from __future__ import absolute_import
+from .graph import DGLGraph
+from ._ffi.object import ObjectBase, register_object
+# from ._ffi.base import c_array
+from . import _api_internal
+from ._ffi.function import _init_api
+from . import backend as F
+
+_init_api("dgl.graph_serialize")
+
+
+@register_object("graph_serialize.GraphData")
+class GraphData(ObjectBase):
+    """GraphData Object"""
+
+    @staticmethod
+    def create(g: DGLGraph):
+        """Create GraphData"""
+        ghandle = g._graph
+        node_tensors = dict()
+        edge_tensors = dict()
+        for key, value in g.ndata.items():
+            node_tensors[key] = F.zerocopy_to_dgl_ndarray(value)
+        for key, value in g.edata.items():
+            edge_tensors[key] = F.zerocopy_to_dgl_ndarray(value)
+        return _CAPI_MakeGraphData(ghandle, node_tensors, edge_tensors)
+
+    def get_graph(self):
+        """Get DGLGraph from GraphData"""
+        ghandle = _CAPI_GDataGraphHandle(self)
+        g = DGLGraph(graph_data=ghandle, readonly=True)
+        node_tensors_items = _CAPI_GDataNodeTensors(self).items()
+        edge_tensors_items = _CAPI_GDataEdgeTensors(self).items()
+        for k, v in node_tensors_items:
+            g.ndata[k] = F.zerocopy_from_dgl_ndarray(_api_internal._ValueGet(v))
```

Comment:
I forgot to expose the `Value` class in python. It is fixed in PR #725 , [here](https://github.com/dmlc/dgl/blob/f3c417c1b000d930196322dd14531efa08f7564a/python/dgl/container.py#L73), which means you could directly use `v.data`.